src/main/java/net/glowstone/net/GlowSession.java

```diff
@@ -375,12 +375,14 @@ public void setPlayer(PlayerProfile profile) {
     }
 
     public ProtocolType getProtocolType() {
-        if (version == GlowServer.PROTOCOL_VERSION)
-            return ProtocolType.PLAY;
-        if (version == GlowServer.LEGACY_PROTOCOL_VERSION)
-            return ProtocolType.PLAY_107;
-        if (version == GlowServer.COMPATIBLE_PROTOCOL_VERSION)
-            return ProtocolType.PLAY_109;
+        switch (version) {
```

Comment:
LEGACY and COMPATIBLE are in my view not the perfect names for the use case. Would go something with more like Bungee does. https://github.com/SpigotMC/BungeeCord/blob/master/protocol/src/main/java/net/md_5/bungee/protocol/ProtocolConstants.java